### DISCUSSION TITLE
which additional raw formats do we need to support?

### DISCUSSION THREAD

**Discussion by lbestock** [arch-kiosk/arch-kiosk-office#1111] [users: ideas for new features]
Canon it appears has a ton of raw formats. #818 took care of .cr2 but there are still CRWs that won't come in and god knows how many others. Do we want to chase this endlessly or write something in a manual that tells how to get new file formats added to the mix? Or is that impossible?

**Comment by urapadmin** [comment]
that is impossible. The file formats are all different. It needs code to interpret a new file format.

**Comment by urapadmin** [comment]
it appears the cr2 format is the more recent format Canon uses. But it is indeed a different file format than crw. 
Before I go and implement a new format I would want to hear if anybody at ustp or on another project is actually using it.
